package.json

```
@@ -0,0 +1,12 @@
+{
+    "scripts": {
+        "doc-prepare": "cd doc && bash doc.sh prepare",
+        "doc-dev": "cd doc && bash doc.sh dev",
+        "doc-build": "cd doc && bash doc.sh build",
+        "doc-build-netlify": "cd doc && bash doc.sh build-netlify",
```

```suggestion
        "doc-netlify": "cd doc && bash doc.sh build-netlify",
```
Keep the same name in all package.json